Source/Core/Core/HW/EXI/EXI_DeviceEthernet.h

```diff
@@ -413,6 +418,60 @@ class CEXIETHERNET : public IEXIDevice
 #endif
   };
 
+  class BuiltInBBAInterface : public NetworkInterface
```

Comment:
The member order in this class doesn't match [our coding style guide](https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#cpp-style-classes-and-structs), so unless there's a specific reason you have them like this please reorder them.